tgstation.dme

```diff
@@ -1925,6 +1925,7 @@
 #include "code\modules\mob\living\simple_animal\bot\medbot.dm"
 #include "code\modules\mob\living\simple_animal\bot\mulebot.dm"
 #include "code\modules\mob\living\simple_animal\bot\secbot.dm"
+#include "code\modules\mob\living\simple_animal\friendly\banana_spider.dm"

```

Comment:
The path in the .dme points to the wrong location. You placed banana_spider.dm in modular_citadel/code/modules/mob/living/, but didn't update the .dme

Reply:
alright, It should be good now